include/gbinder_reader.h

```diff
@@ -142,6 +142,10 @@ gbinder_reader_read_buffer(
     GBinderReader* reader)
     G_GNUC_WARN_UNUSED_RESULT;
 
+const void*
+gbinder_reader_read_parcelable(
+    GBinderReader* reader); /* Since 1.1.XX */
```

Comment:
Shouldn't such function return the size of the parselable, e.g.
```c
const void*
gbinder_reader_read_parcelable(
    GBinderReader* reader,
    gsize* size);
```
The `size` pointer could be NULL if the caller is not interested.

Reply:
I think most of the time the caller would already know the payload size, nonetheless is a good point. Added, thanks!